consumer/fileobservable.go

```diff
@@ -87,10 +87,15 @@ func (fs *observableFile) hasRotated(currentName string) bool {
 
 func (fs *observableFile) storeOffset() {
 	fs.cursor.offset, _ = fs.handle.Seek(0, io.SeekCurrent)
-	offsetAsString := strconv.FormatInt(fs.cursor.offset, 10)
+	fs.saveOffset(fs.cursor.offset)
+}
+
+func (fs *observableFile) saveOffset(offset int64) {
+	offsetAsString := strconv.FormatInt(offset, 10)
 	if err := ioutil.WriteFile(fs.offsetFileName, []byte(offsetAsString), 0644); err != nil {
 		fs.log.WithError(err).Error("Failed to store offset")
 	}
+	fs.log.Info("Offset file reseted")
```

Comment:
I guess this comment should belong to the EOF section of scrape (your change below) as saveOffset might be called with any offset and not just 0.
Besides that - shouldn't it be "Offset file reset"? But my grammar sense might be wrong here 😄

Reply:
You're right. It's better to place it to EOF section of scrape function.
Speaking of message text I wanted to make it somewhat in style of other messages:
"File rotated"
"Offset file reseted"
but if you feel this message wrong I will change it.

Reply:
nvm. I'm not 100% sure what the correct form is, but let's not nit pick about that. I think the message is clear enough.